### PR TITLE
Added great stuff

### PR DESCRIPTION
**Pull request by sharanareva**
This is a cool idea I came up with.  Check it out and let me know.

### CODE REVIEW

File: first.cpp [hunk 1]
```diff
@@ -26,6 +26,10 @@ class Foo
     {
         std::cout << "Printing stuff" << std::endl;
     }
+
+    void greatstuff()
+    {
```
Comment (revadir) [inline first.cpp:31]: I think you may want to add meat to your great stuff? :)

Comment (sharanareva) [inline first.cpp:31]: Oops, yeah, I forgot.  sorry.

File: first.cpp [hunk 2]
```diff
@@ -26,6 +26,11 @@ class Foo
...
+        std::cout << "here you go..." << std::endl;
```
Comment (revadir) [inline first.cpp]: All right, how about this though?
if (everything_is_all_right)
   std::cout << "here you go..." << std::endl;

Comment (sharanareva) [inline first.cpp]: Ok, got it.  Will update.

**Comment by revadir** [comment]
Accepted.

**Pull request merged**